source/util.civet

```diff
@@ -68,6 +68,8 @@ export SourceMap = (sourceString) ->
       mappings: @renderMappings()
       names: []
       sourcesContent: [sourceString]
+      toString: ->
+        JSON.stringify(this)
```

Comment:
Should this be a method on the class or the `json` object? From the failing test, I assume the former, so this should be dedented one level.